app/models/fee_liability.rb

```diff
@@ -21,6 +21,10 @@ def paid?
     govpay_payment_status == 'success'
   end
 
+  def failed?
```

Comment:
To me, "failed? == true" implies that we tried to do something, and we were unsuccessful. As written, are we saying that all fee liabilities start in the failed state, and stay that way until they are paid? If so, maybe we should consider changing our terminology. What do you think?